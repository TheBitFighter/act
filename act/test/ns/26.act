deftype fxp(int x) { }

namespace fxp {

}

