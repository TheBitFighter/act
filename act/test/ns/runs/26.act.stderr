ERROR: File `26.act', line 3, col 10
	Namespace name `fxp' is already used in the current namespace.
